### PR TITLE
provision default grafana datasource

### PR DESCRIPTION
**Pull request by aborilov**

### CODE REVIEW

File: config/grafana/values.yaml [hunk 1]
```diff
@@ -7,4 +7,13 @@ adminPassword: admin
...
\ No newline at end of file
+  storageClassName: standard-v2
+
+datasources:
+  datasources.yaml:
+    apiVersion: 1
+    datasources:
+    - name: Prometheus
+      type: prometheus
+      url: http://thanos-query-frontend.{{ template "grafana.namespace" . }}.svc.cluster.local:9090
```
Comment (rastislavs) [inline config/grafana/values.yaml:18]: This should be available only for superusers... Are we sure that the tenant users will not have access to it with this?

Comment (aborilov) [reply config/grafana/values.yaml:18]: this is only for PoC.
tenant users will have their own organization without this datasource

Comment (jiachengxu) [reply config/grafana/values.yaml:18]: Can we somehow configure the permission of the datasource to make sure that only admin can see this datasource?

Comment (aborilov) [reply config/grafana/values.yaml:18]: as I said, this datasource will be in a different organization. for tenant we will be creating own organization

**Comment by jiachengxu** [comment]
Close this because we are switching to use cortex

**Pull request closed without merging**